pkg/nodeps/values.go

```diff
@@ -175,8 +177,8 @@ func GetValidDatabaseVersions() []string {
 
 // IsValidMariaDBVersion is a helper function to determine if a MariaDB version is valid, returning
 // true if the supplied MariaDB version is valid and false otherwise.
-func IsValidMariaDBVersion(MariaDBVersion string) bool {
-	if _, ok := ValidMariaDBVersions[MariaDBVersion]; !ok {
+func IsValidMariaDBVersion(v string) bool {
+	if _, ok := ValidMariaDBVersions[strings.TrimPrefix(v, MariaDB+":")]; !ok {
```

Comment:
So these were only used in `ddev debug migrate-database` and were always wrong?

Reply:
Yes, that's right.

I didn't even know the validator was broken, the condition in `ddev debug migrate-database` looked strange and I investigated it.